src/services/main/paymentManager.ts

```diff
@@ -27,6 +27,11 @@ interface UserOperationInfo {
     routing_fees?: number
     chain_fees?: number
     confs?: number
+    tx_hash?: string;
+    user_address?: {
```

Comment:
good catch on this one, I never realized the address for received TXs was not being taken in